### PR TITLE
Split requirements for Python2 and Python3

### PR DESCRIPTION
**Pull request by emulbreh**

### CODE REVIEW

File: requirements/shared.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+http://py-monotime.googlecode.com/archive/5fb3eb35a8e25140204e957bd010991bfe8758e3.zip#egg=monotime
```
Comment (mouadino) [inline requirements/shared.txt]: This is only needed for python 2 right ?

Comment (emulbreh) [inline requirements/shared.txt]: right

File: tox.ini [hunk 2]
```diff
@@ -6,9 +6,16 @@ install_command = pip install {opts} {packages}
...
+    -rrequirements/dev.txt
+
+[testenv:py34]
 deps =
     cython==0.20.1
-    -rrequirements/base.txt
+    -rrequirements/base3.txt
     -rrequirements/dev.txt
 
 [testenv:docs]
```
Comment (mamachanko) [inline tox.ini]: we can also remove `cython==0.20.1` from the doc build, can we not?

Comment (emulbreh) [inline tox.ini]: yes

**Comment by mouadino** [comment]
LGTM

**Pull request merged**